.github/workflows/notify-release.yml

```diff
@@ -15,4 +15,4 @@ jobs:
         id: notify
         uses: nearform/github-action-notify-release@main
         with:
-          stale-days: 0
+          notify-after: 0
```

Comment:
does this actually work? I mean how is a number interpreted here? the docs don't say it

Reply:
tested it locally I should add a test